packages/migration-graph-ember/src/entities/ember-app-project-graph.ts

```diff
@@ -0,0 +1,180 @@
+import { relative, resolve } from 'path';
+
+import debug from 'debug';
+import {
+  ProjectGraph,
+  dfs,
+  Package,
+  GraphNode,
+  PackageNode,
+  ModuleNode,
+  Graph,
+} from '@rehearsal/migration-graph-shared';
+import { getInternalPackages } from '../mappings-container';
+import { EmberAppPackage } from './ember-app-package';
+
+const EXCLUDED_PACKAGES = ['test-harness'];
+
+const DEBUG_CALLBACK = debug('rehearsal:ember-app-project-graph');
+
+function debugAnalysis(entry: GraphNode<PackageNode>): void {
+  // Get this list of dependent nodes in some order
+  const dependencies: GraphNode<PackageNode>[] = dfs(entry);
+  const reportedNodes: Set<string> = new Set();
+
+  if (!dependencies || dependencies.length < 1) {
+    return;
+  }
+
+  if (dependencies[0].content.pkg === entry.content.pkg) {
+    return;
+  }
+
+  DEBUG_CALLBACK('List of dependent packages, that need migration:');
+
+  let taskNumber = 1;
+
+  for (const someNode of dependencies) {
+    if (someNode.content.pkg === entry.content.pkg) {
+      break;
+    }
+
+    const packageData = someNode.content;
+
+    const pkg = packageData?.pkg ?? undefined;
+
+    if (!pkg) {
+      continue;
+    }
+
+    const packageName = pkg.packageName ?? '';
+    const duplicate = reportedNodes.has(packageName) ? 'DUPLICATE' : '';
+
+    if (duplicate.length) {
+      DEBUG_CALLBACK('Duplicate found');
+      return;
+    }
+
+    const relativePath = relative(process.cwd(), pkg.path);
+    const parentPkgName = someNode.parent?.content?.pkg?.packageName;
+
+    let taskString = `${taskNumber++}. ${pkg.packageName} (./${relativePath})`;
+
+    if (parentPkgName) {
+      taskString = taskString.concat(` parent: ${parentPkgName}`);
+    }
+
+    if (packageData.converted) {
+      DEBUG_CALLBACK(`[X] DONE ${taskString}`);
```

Comment:
```suggestion
      DEBUG_CALLBACK('[X] DONE %O', taskString);
```

this will pretty print the taskString correctly in terminal